packages/manager/apps/web/client/app/hosting/database/order-private/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,59 @@
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_2_BASES_800_MB": "2 bases de 800 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_50_BASES_100_MB": "50 bases de 100 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_50_BASES_200_MB": "50 bases de 200 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_5_BASES_100_MB": "5 bases de 100 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_5_BASES_200_MB": "5 bases de 200 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_5_BASES_400_MB": "5 bases de 400 Mo",
+  "hosting_database_order_private_domains_pack_value_SQLPERSO_5_BASES_800_MB": "5 bases de 800 Mo",
+  "hosting_database_order_private_hosting": "Hébergements",
+  "hosting_database_order_private_ram": "RAM",
+  "hosting_database_order_private_version": "Version",
+  "hosting_database_order_private_version_value_mariadb_101": "MariaDB 10.1",
+  "hosting_database_order_private_version_value_mariadb_102": "MariaDB 10.2",
+  "hosting_database_order_private_version_value_mysql_41": "MySQL 4.1",
+  "hosting_database_order_private_version_value_mysql_50": "MySQL 5.0",
+  "hosting_database_order_private_version_value_mysql_51": "MySQL 5.1",
+  "hosting_database_order_private_version_value_mysql_55": "MySQL 5.5",
+  "hosting_database_order_private_version_value_mysql_56": "MySQL 5.6",
+  "hosting_database_order_private_version_value_mysql_57": "MySQL 5.7",
+  "hosting_database_order_private_version_value_postgresql_94": "PostgreSQL 9.4",
+  "hosting_database_order_private_version_value_postgresql_95": "PostgreSQL 9.5",
+  "hosting_database_order_private_version_value_postgresql_96": "PostgreSQL 9.6",
+  "hosting_database_order_private_version_value_postgresql_10": "PostgreSQL 10",
+  "hosting_database_order_private_version_value_postgresql_11": "PostgreSQL 11",
+  "hosting_database_order_private_version_value_mongodb_34": "MongoDB 3.4",
+  "hosting_database_order_private_version_value_redis_32": "Redis 3.2",
+  "hosting_database_order_private_version_value_mongodb_40": "MongoDB 4.0",
+  "hosting_database_order_private_version_value_redis_40": "Redis 4.0",
+  "hosting_database_order_private_duration": "Combien de temps désirez-vous l'utiliser ?",
+  "hosting_database_order_private_duration_label": "{{duration}} :",
+  "hosting_database_order_private_payment_total_price_without_tax": "Prix HT :",
+  "hosting_database_order_private_payment_total_price_with_tax": "Prix TTC :",
+  "hosting_database_order_private_error": "Une erreur est survenue lors de la commande d'une base de données SQL: {{message}}",
+  "hosting_database_order_private_success": "Votre <a href=\"{{t0}}\" target=\"_blank\">bon de commande</a> a été généré avec succès. Votre produit sera activé lorsque vous l'aurez validé.",
+  "hosting_database_order_private_success_activation": "Votre activation à été faites avec succès. Votre produit sera activé dans quelques instants."
```

Comment:
```suggestion
  "hosting_database_order_private_success_activation": "Votre activation a été faite avec succès. Votre produit sera activé dans quelques instants."
```

Reply:
Good catch. I'll check if it's not anywhere else (sentence was copy paste)